### PR TITLE
Parax nft

### PR DESCRIPTION
**Pull request by zhoujia6139**
## Security Checklist

- [ ] 1. Re-Entrancy
- [ ] 2. Arithmetic Over/Under Flows
- [ ] 3. Unexpected Ether
- [ ] 4. Delegatecall
- [ ] 5. Default Visibilities
- [ ] 6. Entropy Illusion
- [ ] 7. External Contract Referencing
- [ ] 8. Short Address/Parameter Attack (off chain)
- [ ] 9. Unchecked CALL Return Values
- [ ] 10. Race Conditions / Front Running
- [ ] 11. Denial Of Service (DOS)
- [ ] 12. Block Timestamp Manipulation
- [ ] 13. Constructors with Care
- [ ] 14. Uninitialized Storage Pointers
- [ ] 15. Floating Points and Precision
- [ ] 16. Tx.Origin Authentication
- [ ] 17. Address.isContract Re-Entrancy via Constructor

### ⚠️ NOTES ⚠️

Make sure to think about each of these exploits in this PR.

### CODE REVIEW

**Review by GopherJ — DISMISSED** [review]

File: helpers/contracts-deployments.ts [hunk 1]
```diff
@@ -3276,6 +3278,37 @@ export const deployDelegationAwarePToken = async (
...
+  const initData = implementation.interface.encodeFunctionData("initialize");
+
+  const proxyInstance = await withSaveAndVerify(
+    await getContractFactory("InitializableAdminUpgradeabilityProxy"),
+    eContractid.ParaX,
+    [],
+    verify
+  );
+
+  const proxyAdmin = "0x17816E9A858b161c3E37016D139cf618056CaCD4"; //yubo.eth
```
Comment (GopherJ) [inline helpers/contracts-deployments.ts:3293]: seems hard coded

Comment (GopherJ) [reply helpers/contracts-deployments.ts:3293]: might better be our safe address? 0x19293FBec52F94165f903708a74513Dd6dFedd0a

File: test/_parax_spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    await waitForTx(await ParaX.transferOwnership(user1.address));
+    await waitForTx(
+      await ParaX.connect(user1.signer).mint([user1.address, user2.address])
+    );
+    expect(await ParaX.ownerOf(2)).to.be.eq(user1.address);
+    expect(await ParaX.ownerOf(3)).to.be.eq(user2.address);
+    expect(await ParaX.totalSupply()).to.be.eq(4);
+
+    expect(await getProxyAdmin(ParaX.address)).to.be.eq(
+      "0x17816E9A858b161c3E37016D139cf618056CaCD4"
```
Comment (GopherJ) [inline test/_parax_spec.ts:36]: same here

File: helpers/contracts-deployments.ts (hunk 1 above)
Comment (zhoujia6139) [reply helpers/contracts-deployments.ts:3293]: It's a one-time deploy contract and yubo ask use his addrss as proxy admin

**Pull request closed without merging**